semver.py

```diff
@@ -164,6 +164,74 @@ def __iter__(self):
         for v in self._astuple():
             yield v
 
+    def bump_major(self):
+        """Raise the major part of the version
+
+        :return: the raised version string
```

Comment:
maybe "new object with raised major part" would be better? Just to indicate that after calling this function new VersionInfo object is returned, not text string. I know that return type is already described below but I don't know how to do it better :)

Reply:
I absolutely agree, it was badly worded. :wink: This part is crucial and needs to be documented.

Will change that.